### DISCUSSION TITLE
Disabling hard wraps for HTML files?

### DISCUSSION THREAD

**Discussion by superDuperCyberTechno** [helix-editor/helix#9846] [Q&A] [answered]
I'm currently using `vscode-html-language-server` and I'm slowly going crazy from the constant hard wraps the language server is doing at line 80 (I'm assuming).

Does anyone know how to disable this option? I've tried the following (quite naively) as I have no idea how to take this on.

```
#languages.toml
[language-server.vscode-html-language-server]
config = { "html.format.wrapLineLength" = 0 }
```

**Comment by David-Else** [comment, marked as answer]
You could use Prettier to format instead, if you don't like the defaults you can probably change them:

```
[[language]]
name = "html"
formatter = { command = 'prettier', args = ["--parser", "html"] }
```

You can probably change the `vscode-html-language-server` using config, I had a quick look to try and find the commands but they are not properly documented. Most people are using them inside VS Code so it is a bit niche to call them manually in a Helix config. A good place to look for config info is in the Neovim setups, for example: https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#html . This says you could turn off the built in formatting with `provideFormatter = false`.

**Comment by superDuperCyberTechno** [comment]
Thanks, man. This did the trick:

```
[[language]]
name = "html"
formatter = { command = 'prettier', args = ["--parser", "html", "--print-width", "9999"] }
```
